harness/determined/common/api/authentication.py

```diff
@@ -215,6 +215,9 @@ def logout(
     if session_user is None:
         return
 
+    if session_user == token_store.get_active_user():
```

Comment:
Extra credit 1:`logout` is too complicated to not have a docstring that describes its semantics.

Reply:
ok I extended the docstring.